### DISCUSSION TITLE
公私分明？社交冷感？

### DISCUSSION THREAD

**Discussion by vchien03** [1101-GS4719-C/Python-Community-GS4719-C#1873] [General]
[辦公室異常安靜！課長困惑：年輕員工像得自閉症 網友一面倒曝心聲](https://tw.news.yahoo.com/%E8%BE%A6%E5%85%AC%E5%AE%A4%E7%95%B0%E5%B8%B8%E5%AE%89%E9%9D%9C-%E8%AA%B2%E9%95%B7%E5%9B%B0%E6%83%91-%E5%B9%B4%E8%BC%95%E5%93%A1%E5%B7%A5%E5%83%8F%E5%BE%97%E8%87%AA%E9%96%89%E7%97%87-%E7%B6%B2%E5%8F%8B-%E9%9D%A2%E5%80%92%E6%9B%9D%E5%BF%83%E8%81%B2-012127589.html)
類似的新聞越來越多，現在人們好像更傾向於享受屬於自己的個人時間，而不是與他人社交。

不過這篇新聞主要是指工作場所，之前就聽過有人覺得工作就是工作，同事、朋友關係全部混在一起反而讓人混亂，下班還要見到同一群人就感覺像還沒下班，所以認為要把同事關係跟私生活完全分開。
不知道大家有甚麼看法呢？

**Comment by 12Jesse24** [comment]
我的話可能會看人，同事是否等於朋友，如果成立那我覺得這種關係(既是朋友也是同事)是可以維持的。如果同事相處起來很不ok的話我就會傾向於把同事跟朋友分開來。但其實能不能朋友很看感覺XD

**Comment by sorn32** [comment]
我覺得至少上班跟下班的狀態要分開，上班是同事，下班是朋友，這樣比較不會有混亂的感覺。

**Comment by FakeWood** [comment]
我覺得這也是時代的差異之一，聽上一輩的人講，他們讀書的時候還會去舞廳跳舞聯誼之類的，感覺以前人與人之間的聯繫比較容易與緊密，我覺得這可能是我們現在都強調自我個性與個體發展所帶來的結果吧!

**Reply by vchien03** [reply]
我同意~看到長輩與朋友之間的相處熱絡，感覺真的跟我們這一代很不一樣。現在真的很強調自我個性，也會看到像Me time這一類型的東西出現。

**Comment by RavenJiaSin** [comment]
我認為若把社交當成是一種不得不做的任務的話確實會感到疲憊，但如真的把這群人當成好朋友的話，我認為和好朋友相處應該是不會感覺到疲憊的吧?

**Comment by vchien03** [comment]
我認為這樣的情況也跟科技發展有關。以前科技還沒那麼進步，所有事情只能通過見面或書信來傳達，因此人與人之間必然會互相接觸。但現在只要動動手指就可以在網路上找到大量資訊，就減少了很多與人相處的次數了。

**Comment by chuanmin0317** [comment]
現代的生活娛樂不像以往那樣需要面對面才能交流，像是網路遊戲就能讓我們在不同的地區卻一起互動，也有許多活動能夠讓我們享受自己的休閒時光，所以我想人與人之間的疏離也是必然的

**Comment by YANRONG0123** [comment]
我覺得要看私底下是否能和同事擁有相同興趣來判斷，畢竟幾乎所有人上班都是為了賺錢過上好生活，進到工作場合就是為了能夠完成自己所被分配到的任務，且能夠準時下班做自己想要做的事情，除非公司願意給時間自行運用，不然還是公私分明會比較好。

**Comment by YuanWang08** [comment]
我覺得上班並不是每個人本來就很有意願的事，就像上學一樣，有時只是迫不得已坐在那裡，情緒自然就比較消沉不說話顯得安靜。還有一點我覺得是時空背景不同了，以前做辦公室的工作大多不錯，員工自然有較高的活動力。現在辦公室工作隨處可見，而且薪資待遇無法滿足需求，所以促使了這種現象。

**Comment by LinChloe** [comment]
我覺得跟同事當朋友對我來說有點困難。
下班後再看見他們就感覺自己還沒有下班。
而且如果沒交情以後辭職不幹，比較不會難過。

但我覺得是否和同事交朋友和公司氛圍也有關係。
但我西為增加感情不要是下班後續約，我只想回家睡覺而已。

**Comment by yxin214** [comment]
我能理解文章中員工的想法！工作時間就好好工作，有必要溝通的時候再講話就好了，不需要把工作場合用得和樂融融。我認為同事就是一起工作的人，若有幸變成朋友也不錯，但若雙方頻率不合就不用互相干涉，各自完成分內之事就好。

**Comment by tzurou** [comment]
我自己覺得很看人，有些同事頻率對了下班以後就自然會是朋友，頻率不對那就好好當同事，但是上班期間個人覺得大家都要先做好自己的工作，頂多休息時間在茶水間小聊，下班後有的是時間玩樂聊天。

**Comment by YifenLin** [comment]
其實我有想過未來我進入職場會希望辦公室的氣氛如何，雖然距離踏入社會可能還需要兩三年的時間，但我目前的想像是希望像是youtube 頻道"Decard video"呈現給人的感覺，同事之間彼此還是有話聊，工作之餘也會一起在下班從事娛樂活動，不知道科技業的辦公室會不會有這樣的氛圍存在。不過也許我之後會有不同的想法，畢竟兩種情形都各有優缺，像是若跟同事感情太好，如果要換工作甚麼的也會有人情壓力。

**Comment by happy246810** [comment]
我覺得同事是否為朋友，這要看個人的想法，同時我覺得工作性質也會有差，如果是設計部門之類的，當然越熟識越好，比較可以有討論的熱度。而如果再一些莊嚴場合工作的人，當然要把工作跟私生活完全分開。

**Comment by Hahahahbla** [comment]
現代人對於工作和個人生活有不同的需求。有些人更傾向於將工作與私人生活完全分開，以保持心理上的平衡和清晰。但也有人認為同事間的良好關係可以提升工作效率和樂趣。我認為彼此之間的尊重和理解很重要，而是否成為朋友應該取決於個人意願和相處方式。

**Comment by yyujimmy** [comment]
這種現象確實在現代社會越來越普遍。人們可能更傾向於在工作場所保持專注，並將個人生活與工作分隔開來。這反映了現代社會中對於工作和生活平衡的重視，以及對於個人隱私和空間的尊重。

對於一些人來說，將工作與個人生活分開可能有助於提高工作效率和專注度。他們可能認為，在工作場所保持一定的距離和專業性，有助於維持良好的工作關係和效率。此外，一些人可能也更喜歡在工作之外尋找社交和娛樂的機會，而不是與同事深度交往。

然而，這種現象也可能導致工作場所的氛圍變得冷漠和缺乏凝聚力。良好的團隊合作和人際關係對於工作的順利進行和團隊的成功至關重要。因此，對於某些職場來說，適度的社交和團隊活動可能有助於增強團隊凝聚力和合作性。

總的來說，這是一個需要平衡的問題，不同的人有不同的偏好和需求。重要的是要尊重每個人的選擇，並在工作場所中創造一個既專業又有人性化的環境。

**Comment by jim1743** [comment]
我覺得在工作上，要把同事當朋友會有難度，我個人不是恨善於社交，能正常與同事進行工作上的互動已經很盡力了，自己比較喜歡享有個人的時間，若要下班後還去勉強自己與同事交朋友，對我來說是很痛苦的事。

**Comment by wanka2030** [comment]
在大學生活中，社交冷感可能對學生的學業和社交活動產生不利影響。這些學生可能會感到孤立，避免參加社交聚會或小組討論，從而錯失建立人際網絡和合作學習的機會。社交冷感也可能導致心理壓力增加，影響學業表現和整體幸福感。大學應提供心理輔導和支持，鼓勵社交冷感的學生逐步參與社交活動，並建立安全的環境，以促進他們的社交和情感發展。

**Comment by thomasxshen** [comment]
我認為這是很正常的行為，現代生活特別忙碌，上班的時候就是個同事而已，而對於同事是沒必要敞開心胸的，畢竟知人知面不知心，你的同事其實也是你的潛在競爭對手，那天他可能跑來搞你你也不知道。

**Comment by Mitochondrionjessica** [comment]
我自己也傾向公私分明，才不會在公事上因欠人情而無法公事公辦。

**Comment by cassy0201** [comment]
我覺得能遇到變朋友的同事很難得，若無法成為朋友就當普通同事也不錯，不要交惡就好

**Comment by 710388** [comment]
很正常吧，因為現在常常過度社交，在一段時間後本來就會有冷感的現象產生

**Comment by NcuJerryKao** [comment]
我理解課長的困惑，但也必須看到現代工作環境和社交習慣的變化。許多年輕人更重視工作和私人生活的平衡，他們希望在工作時間專注工作，下班後享受自己的時間，而不是延續職場的人際關係 。

以下是一些觀點和建議：

工作與生活分開：許多人認為同事之間保持專業距離有助於工作效率和個人生活質量 。這樣可以避免在工作外也感到壓力，享受更多私人空間。

文化差異：不同世代對於工作環境中的社交互動有不同的期望。年輕一代可能更偏好使用數位工具進行溝通，避免面對面的社交壓力 。

鼓勵適度互動：雖然維持專業距離很重要，但也可以適度鼓勵團隊活動，增加彼此的了解和協作感。例如，舉辦午餐會或團隊建設活動，讓員工在輕鬆的環境下互動 。

尊重個人選擇：每個人對於社交的需求不同，尊重個人選擇非常重要。企業可以提供多樣化的選擇，讓員工自行決定參與程度 。

總的來說，適應現代職場的變化，理解並尊重不同員工的需求和偏好，能夠創造一個更和諧且高效的工作環境。

**Comment by yubo0oo** [comment]
我很享受一個人認真做事工作的時間
我不喜歡與人交流

**Comment by poiu6fi** [comment]
如果我的同事是朋友的話，那我在工作的時候還是會跟她講一些垃圾話

**Comment by buoioiry** [comment]
我覺得一群人一起工作的話可能可以在裡面找到幾個適合當朋友的人，不會特別避諱跟同事當朋友，不過還是會爭取自己可以獨處的時間，像放假這種時候就會比較想跟家人或是其他朋友相處，畢竟如果跟同事是朋友，在辦公室就已經可以講奇怪的話了，不會特別想多花時間出門玩
